components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -35,6 +35,9 @@
 ! variables and outputs expected in E3SM.                                                  !
 !__________________________________________________________________________________________!
 
+#define bfb_square(val) (val)*(val)
```

Comment:
Here you should wrap everything in `()` to get the association right in the larger expression: `((val)*(val))`.